app/views/posts/search.html.haml

```diff
@@ -0,0 +1,21 @@
+- content_for :title do
+  = @title = "#{t(:find_posts_for)} #{@parent.journal_display_name}"
```

Comment:
Translators will not like this b/c they can't control the position of the journal name, they won't be able to control for journals that begin with a vowel or consonant, and they won't be able to control for gender of the journal name. Here's an alternative that skirts two of these issues:

```ruby
= @title = t( :find_posts_for, vow_or_con: @parent.journal_display_name[0].downcase, journal_name: @parent.journal_display_name )
```

And then in en.yml:

```yaml
en:
  find_posts_for: "Find Posts for: %{journal_name}"
```

Gender could still be a problem, but there's no perfect way to deal with that with arbitrary journal names.

Reply:
Excellent call, and a great lesson for thinking in i18n